tests/integration/AutomaticEmails/WooCommerce/Events/AbandonedCartPageVisitTrackerTest.php

```diff
@@ -28,6 +28,7 @@ public function _before() {
     $this->currentTime = Carbon::now();
     Carbon::setTestNow($this->currentTime);
 
+    // @phpstan-ignore-next-line
```

Comment:
There was a problem:
`Property MailPoet\AutomaticEmails\WooCommerce\Events\AbandonedCartPageVisitTrackerTest::$wp (MailPoet\WP\Functions&PHPUnit\Framework\MockObject\MockObject) does not accept MailPoet\WP\Functions. `
I have now tried a store output of the function `makeEmpty` in a variable and annotate it. And it has helped. I had no idea before because the function `makeEmpty` is from the Codeception library.